index.js

```diff
@@ -183,10 +183,13 @@
          */
         render: function() {
           var passedProps = this.props;
-          var props = { ref: 'instance' };
+          var props = {};
           Object.keys(this.props).forEach(function(key) {
             props[key] = passedProps[key];
           });
+          props.ref = 'instance';
+          props.disableOnClickOutside = this.disableOnClickOutside;
+          props.enableOnClickOutside = this.enableOnClickOutside;
```

Comment:
Actually, we can't, at least for disableOnClickOutside. If you pass a prop of that name to the wrapped component, it's the default disable. I'm not a huge fan of them having the same name, but it would be a major backward compatibility break to change that now, and it doesn't have a significant benefit to do so that I can see.